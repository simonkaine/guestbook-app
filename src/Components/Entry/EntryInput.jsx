import React from 'react'

export default function EntryInput({entry: {name, message}}) {
    return (
        <div>
            <h1>{name}</h1>
            <p>{message}</p>
        </div>
    )
}
