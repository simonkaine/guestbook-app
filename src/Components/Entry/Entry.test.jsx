import React from 'react'
import { render } from '@testing-library/react';
import EntryInput from './EntryInput';

it('Should render a user input', () => {
    const {container} = render(
        <EntryInput entry={{ name: 'Simon', message: 'I was here!' }}/>
    )
    expect(container).toMatchSnapshot(); 
})
