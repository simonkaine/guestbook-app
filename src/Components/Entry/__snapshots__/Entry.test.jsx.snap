// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render a user input 1`] = `
<div>
  <div>
    <h1>
      Simon
    </h1>
    <p>
      I was here!
    </p>
  </div>
</div>
`;
